packages/core/src/utils/viaProxy.test.ts

```diff
@@ -0,0 +1,64 @@
ECZkQImBEhYEaEgBkRAmZECJgRIWBGhIAZEQJmRAiYESFgRoSAGRECZkQImBEhYEaEgBkRAmZECJgRIWBGhIAZEQJmRAiYESFgRoSAGRECZkQImBEhYEaEgBkRAmZECJgRIWBGhIAZEQJmRAiYESFgRoSAGRECZkQImBEhYEaEgBkRAmZECJgRIWBGhIAZEQJmRAiYESFgRoSAGRECZkQImBEhYEaEgBkRAmZECJhlu3+AdNE/Xw+P1+yhGlmqxmZ9cE4bP9DKav33bLuH5GRp0RjNH6VxfZQb1bHzeuOofr1P1e0ckur6ayDVqUOMkIZTp5uIVjS7fwa/O67Xq9M1oKD1/ktN+skOPbU7wSH98vX6TN3av/X+i436/mY9/+7nXX8NpDp1iBHScOo0RIQaXqJdc9U7t90FD7yp1fGnS1ZEVbM0ZWDi9Q1NunuD/nE8/PprINWpQ4yQhlOnJ94T6smJwbm49UPNqNR1z6vXak1+TeveDxY8cZty4n9PC0YH59b5Bj28VWUvqOh3+tp67TglSdlZ+s1kZUXCr78GUp06xAhpOHV6yvRnwrJCHf22ohHtOKUpf9bFxti3siL6672qKJek8a9o3+nYt/bO07i+kjRrozYcie0vyNabs3TbAEm6d6Mqj4Rc391CTB1ihHSbOm1l+jNhRbmiEUn6xe64c1FSU7P+fizYzo/G9t/YKzi3qmrizi1JFxq0dFuw/c0RIddfA6lOHWKENJw6bWX6p6NrDmjNgcTfys7SzCGSdKZe+8/E9k++PthYeyjBUVs/VE2dygs1rTzk+vaUFWrfPPXLl6Sp67Qp/t3UqFLtmaeCbJ2p17g/qqauo4dKdeoQI3TV1Jkg058J24pGNKhIC0Zr6xxNLpOkx3aoriG2YEyfYOPKe5u2dn0kSUOKVZwTZn17TtRpyZZg+5mvxr1hi0irpqogW5KWbElSYEIdTx1ihK6aOhNk+jNhK7Nu0GszYp8WfHxRP96uVdVxawYVBRsHPk38IIda9pcX6t1PUl7fgT/8T/cd1LdGamwfLR2vJ1uuIiy+WVMHBgtePtjRIySUdOoQI3Th1D0ez4RxyoviPq/75JJKclp/NNo3L9g4U5/4Qc5eCjZ654VZ37ElW3S8TpJ+OkFDiiVpSLGWTZSk43X6webkj9BW0qlDjNC1U/dsRBinT552faSqGh2ulaQRpXr6dr0+U7mf+T3lRSWpvlFN7XyufKHlVVxeVpj1HTtdr4WbJKkoWyvvVF5UL1WoJEeSFm7S6XbO+I4lnTrECF07dc+W8b+AeMv+owlrVbFew1/S3Rt09LwkTRukxyfE1jQ2SVJnLuw0hVqfVOURPVctSfcM0b/nBm/hnq0O/1l/0qlDjNDlU/dgRNiuvx3TjEpdapKkH44N/rRLOtcgSflRtXeduaDljXZ9Y5j1nbF0W/CsNaa3JB2q1SPbOjyg0xJOHWKE7pi6pyLCjuw/o1cPSVKvXH2pX7CzNtmbmav3gl15cZjq+s6ovayH3op9uXSbzl3u7LFJtZ06xAjdMXVPRYRJvHM22ChrudH5/XPBxrCSxIcML5WkZulEXZj1nXT/yNj2925K4cDOaDV1iBG6aeoeiQiTuPoH+3zLBwlvt1zC/sp1iQ+5sv9wbXCdLdX1nfGNYfrOjbEvZw/VgtGdPbYzWk0dYoTumLqnyvQIX6xQ8yI1PJj4D3Y0ollDg+33Wq5l/etksHHf8ASHTC5TWaEkvXUi5PqkBhRoxRRJutSk71YF799+dYcGF3V8XEyqU4cYocun7sEyPcLNJyQpGtGyiQnu6H/iVo0qlaTdH+tIy+urw7XaeUqS7hmiOcPi1hdk6+lJwfbV27VSXZ/UyjvVP1+SfrZTL7ynx3dKUq9cPTe1s4+Q6tQhRujyqXuwTI/w5QPB1eT7R6hypirK1TdP+VFNGqC10/XoLcGyn++KO+q3bwcbv5+mx76s8kLlRzVlYOyfAw5+qnVHwq/vwAM3Bef0P08Gd8w8tVvbT0rS9MFafHN3TR1ihC6cumfL9H9lkjR/lF6s6GjBqmo9uCluT0TaPDu4QNdWszSzUn85Gn59e24o1t55Ks3VhQbd8qfY3V5jemvXXOVHde6yvviKDtUmf6hUpw4xQldN3eNl+jOhpDUHtHBT6//ouaJZWr5Xi9vcC9Ysff2N4BbkVhqatHhz63Mr1fUJRaTVd6k0V5Ie3R53v+U7Z4MXpcU5Wn1Xu5fmPivVqUOM0CVTZwKeCQPDS/TQOM0YrGElyoqo5ryqavTM/sTn0BXZWVr0Bc0fpbF9VZStE3WqqtEv92jP6a5Z38qPxmv57ZJUVaNp61vfjBKNaMscTRogSY9s0/K9nXrMVKcOMcLnnDoTECFgxstRwIwIATMiBMyIEDAjQsCMCAEzIgTMiBAwI0LAjAgBMyIEzIgQMCNCwIwIATMiBMyIEDAjQsCMCAEzIgTMiBAwI0LAjAgBMyIEzIgQMCNCwIwIATMiBMyIEDAjQsCMCAEzIgTMiBAwI0LAjAgBMyIEzIgQMCNCwIwIATMiBMyIEDAjQsCMCAEzIgTMiBAwI0LAjAgBMyIEzIgQMCNCwIwIATMiBMyIEDAjQsCMCAEzIgTMiBAwI0LAjAgBMyIEzIgQMCNCwIwIATMiBMyIEDAjQsCMCAEzIgTMiBAwI0LAjAgBMyIEzIgQMCNCwIwIATMiBMyIEDAjQsCMCAEzIgTMiBAwI0LAjAgBMyIEzIgQMCNCwIwIATMiBMyIEDAjQsCMCAEzIgTMiBAwI0LAjAgBMyIEzIgQMCNCwIwIATMiBMyIEDAjQsCMCAEzIgTMiBAwI0LAjAgBMyIEzIgQMCNCwIwIATMiBMyIEDAjQsCMCAEzIgTMiBAwI0LAjAgBMyIEzIgQMCNCwIwIATMiBMyIEDAjQsCMCAEzIgTMiBAwI0LAjAgBMyIEzIgQMCNCwIwIATMiBMz+Dzt6qG+Jr4RtAAAAAElFTkSuQmCC';
```

Comment:
Should I trim this? 

I dont think the actual payload is relevant for the unit test.

Reply:
I'd say just put in a smaller image, i.e. 1x1 pixels

Reply:
I must have accidentally deleted my comment earlier on this: you could keep it and replace it with a 1x1 image so it's not so long

Reply:
oh… good point!